bookshelf.js

```diff
@@ -0,0 +1,80 @@
+/* Button create and click listener */
+var button = document.createElement("button"); // create the button
+button.setAttribute("id", "fetch-books-btn"); // set id attribute to the button
+var buttonText = document.createTextNode("Fetch Books"); //create the button text
+button.appendChild(buttonText); // apply buttonText to Button tag
+button.addEventListener("click", fetchBooks); // click event trigger
+document.body.appendChild(button); //append button to the body element
+
+/* fetchBooks function */
+
+function fetchBooks() {
+	const booksJSON = 'https://raw.githubusercontent.com/codeyourfuture/bookshelf-project/master/books.json'
+	fetch(booksJSON)
+    .then(response => response.json())
+    .then(processBooks) // call the processBooks function
+    .then(removeBtn) // call the removeBtn function
+}
+
+function processBooks(booksJSON) {
+	var ulTag = document.createElement("ul"); // create <ul> Tag
+
+	// This forEach loop creates several li tags
+	booksJSON.forEach(function(el){
```

Comment:
Nitpick: I think `book` would be a better name for this variable

Reply:
done!